### DISCUSSION TITLE
Possibility to change video canvas background colour.

### DISCUSSION THREAD

**Discussion by chlowden** [smplayer-dev/smplayer#927] [Ideas]
Hello
I am looking for general purpose video player that has the ability to change the video canvas window background colour to any colour.
Why ?
To check for edge video blanking issues that are often black.  Changing the colour will show up these sorts of encoding errors far more easily.
Thank you for a great video player.

**Comment by Raza-0007** [comment]
You can do this in mpv, but I don't think it is possible with smplayer, because smplayer draws its own background, and overrides any mpv option.

Here are the instructions for Windows 11/10. I don't know if they will work in Linux.

You will need to create a custom text file `mpv.conf ` inside the mpv folder `smplayer\mpv\mpv\mpv.conf`

Open `mpv.conf` in notepad and paste this option

`background-color=1.0/0.0/0.0`

This will make the background color red

Run mpv directly from `smplayer\mpv\mpv.exe`

~Raza

**Reply by chlowden** [reply]
Hello
Thank you for your reply. I am using Rocky Linux 8.9. my SMPLAYER is installed through flatpak and sadly, I cannot find an mpv.conf folder in the multitude of SMPLAYER folders.

I also installed MPV directly ... Note for Centos / Rocky users, you need to enable powertools 
`sudo dnf config-manager --enable powertools.`
`sudo dnf install mpv`

 I added  `background-color=1.0/0.0/0.0` to the end of the mpv.conf
`sudo nano /usr/share/doc/mpv/mpv.conf`

I also tried 
`sudo nano ~/.var/app/io.mpv.Mpv/config/mpv/mpv.conf`

But sadly that has not changed the background.

**Comment by smplayer-dev** [comment]
In order to change the background color in smplayer, you need to change [this line](https://github.com/smplayer-dev/smplayer/blob/master/src/basegui.cpp#L2553) in the source code and recompile:

`panel->setStyleSheet("#panel { background-color: black; }");`
